mapie/estimator/interface.py

```diff
@@ -37,4 +37,18 @@ def predict(
         """
         Predict target from X. It also computes the prediction per train sample
         for each test sample according to ``self.method``.
+
+        Parameters
+        ----------
+        X: ArrayLike of shape (n_samples, n_features)
+            Test data.
+
+        **kwargs : dict
+            Additional fit and predict parameters.
+
+        Returns
+        -------
+        Tuple[NDArray, NDArray]
+            - Predictions
+            - Predictions sets
```

Comment:
I just realized that your PR does not concern this part of the code (and which will conflict with another PR in progress). So it would be safer not to add these changes.
```suggestion
```

Reply:
What do you suggest to do ? @thibaultcordier